androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/TripStops.kt

```diff
@@ -101,7 +101,7 @@ fun TripStops(
     Box {
         Box(
             Modifier.matchParentSize()
-                .padding(4.dp)
+                .padding(horizontal = 4.dp)
```

Comment:
This is a mistake I fixed more thoroughly in #909 - the padding on the bottom is only correct if there’s an alert card at the bottom of the list, but since there can’t be an alert card in the list as of this PR, just never having the padding at the bottom is the right fix for now.